dist/api_data.py

```diff
@@ -983,6 +983,12 @@ def __ge__(self, other):
         size and the default config would extend log files in allocations of
         the maximum log file size.''',
         type='list', choices=['data', 'log']),
+    Config('file_handle_close_sync', 'true', r'''
```

Comment:
While I'm not involved in the overall discussion, my one comment here would be to shorten the API name. This would be one of the longest names otherwise. I think that `file_close_sync` is equally clear. If you make that change please rename the WT_CONN flag also.

Reply:
Fixed